### PR TITLE
Cleanup /etc/motd after doc link shift

### PR DESCRIPTION
**Pull request by zmerlynn**
Fit and finish logging into a 1.2-beta.1 node now.

### CODE REVIEW

**Comment by eparis** [comment]
This PR has added the `cherrypick-candidate` label but did not set a milestone. This is an invalid state and will not be considered for cherrypick-ing to any release branch. The `cherrypick-candidate` label will be automatically removed from such PRs in the future.

File: cluster/gce/configure-vm.sh [hunk 1]
```diff
@@ -95,8 +93,8 @@ If it isn't, the closest tag is at:
 
 Welcome to Kubernetes ${version}!
 
-You can find documentation for this release at:
-  http://kubernetes.io/${minor}/
+You can find documentation for Kubernetes at:
+  http://kubernetes.io/
```
Comment (roberthbailey) [inline cluster/gce/configure-vm.sh]: Maybe docs.kubernetes.io/ instead?

Comment (zmerlynn) [inline cluster/gce/configure-vm.sh]: Done

**Comment by zmerlynn** [comment]
@k8s-bot test this, issue #22671

**Comment by bgrant0607** [comment]
@k8s-bot unit test this issue: #17998

**Comment by bgrant0607** [comment]
@k8s-bot unit test this issue: #17998

**Pull request merged**